source/includes/_issuers.md

```diff
@@ -70,25 +76,31 @@ Content-Type: application/json
 
 Create a new Issuer.
 
-Attribute | Description
---------- | -----------
-**created** | Created is date generated when Issuer created. Default value `now date`: `timestamp`.
-**modified** | Date generated when Issuer is updated. Default value `now date`: `timestamp`.
-**active** | Active is `boolean`. Default `true`.
-**deleted** | Deleted is `boolean`. Default `false`.
-**verified** | Verified is `boolean`. Default `false`.
-**blacklisted** | Blacklisted is `boolean`. Default `false`.
-**account_number** | Account_number is `string`, must contain 11 digits.
-comments | Comments. `string`.
-name | Name of Issuer. `string`.
-logo_url | The URL of the Issuer's logo. `string`.
-address | Issuer address. As an [`Address`](#address) object.
-is_messaging_mandatory | Determines if *message* field is mandatory for invoices issued by this issuer. `boolean`. Default value `false`.
+Arguments | Type | Description
+--------- | ---- | ------
+**created** | `timestamp` | Created is date generated when Issuer created. _default value `now date`_
+**modified** | `timestamp` | Date generated when Issuer is updated. _default value `now date`_
+**active** | `boolean` | Flag that sets Issuer object to active. _default `true`_
+**deleted** | `boolean` | Flag that sets Issuer object to deleted. _default `false`_
+**verified** | `boolean` | Flag that sets Issuer object to verified. _default `false`_
+**blacklisted** | `boolean` | Flag that sets Issuer object to blacklisted. _default `false`_
```

Comment:
Are you sure we should have these fields? From created to blacklisted. These are automatically set in the API. I think it's better to explain fields like these in the method object introduction.